src/packages.builds

```diff
@@ -33,6 +39,10 @@
 
   <ItemGroup Condition="'$(BuildAllPackages)' == 'false' AND '$(SkipNativePackageBuild)' != 'true'" >   
     <!-- add specific native builds / pkgproj's here to include in servicing builds -->
+    <Project Include="Native\pkg\**\*.builds">
+      <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
```

Comment:
You need to set `SkipCreatePackageOnMissingFiles=true` (see the build all section above which does this).

Reply:
Thanks, updated.